### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by xerces13** [Chia-Network/chia-blockchain#5319] [Support]
SYNC IMPOSSIBLE

I try everything, when the last block arrive PAF! NOT SYNCED! I have updated the GUI to the latest version. I add peer. I opened the port 8444,8449,8447 and I checked via checker that they are open. I use openVPN, not synced. I use Vpn cyberghost. I made the necessary redirection on my router. I have deactivated firewall on my computer. Honestly, I am thinking of giving up CHIA MINING ..

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
